docs/sources/whatsnew/whats-new-next/index.md

```diff
@@ -16,3 +16,16 @@ weight: -37
 # What’s new in Grafana Cloud
 
 Welcome to Grafana Cloud! Read on to learn about the newest changes to Grafana Cloud.
+
+## Support query-type template variables for Tempo data source
+
+<!-- Fabrizio Casati -->
+<!-- OSS, Enterprise -->
+
+_Generally available in Grafana Cloud_
+
+The Tempo data source now supports query-type template variables. With this, you can now create variables whose values are a list of attribute names or attribute values seen on spans received by Tempo.
+
+To learn more, refer to the video demo below as well as to the [Variables page](https://grafana.com/docs/grafana/latest/dashboards/variables/) in the Grafana documentation.
```

Comment:
Avoiding above/below for improved accessibility and tweaking the wording for docs link and using a partial URL
```suggestion
To learn more, refer to the following video demo, as well as the [Grafana Variables documentation](/docs/grafana/latest/dashboards/variables/).
```